cfme/tests/openstack/infrastructure/test_provider.py

```diff
@@ -19,14 +19,13 @@ def test_api_port(provider):
 
 
 def test_credentials_quads(provider):
-    navigate_to(provider, 'All')
-    quad = Quadicon(provider.name, qtype='infra_prov')
-    checked = str(quad.creds).split('-')[0]
-    assert checked == 'checkmark'
+    view = navigate_to(provider, 'All')
+    prov_item = view.items.get_item(by_name=provider.name)
+    assert 'checkmark' in prov_item.data['creds']
```

Comment:
**Optional**
Consider `assert prov_item.data.get('creds') and 'checkmark' in prov_item.data['creds']`